pythonPackages.nix

```diff
@@ -7,4 +7,18 @@ in {
   pyscf = callPackage ./pyscf { };
   pyquante = callPackage ./pyquante { };
   pychemps2 = callPackage ./chemps2/PyChemMPS2.nix { };
+  qcelemental = callPackage ./qcelemental { };
+  qcengine = callPackage ./qcengine { };
+  gau2grid-1_3_1 = callPackage ./gau2grid { version = "1.3.1"; sha256 = "0zkfil7cxjip79wqvhljk1ifjq0cwxzx6wlxgp63b6wbagma0i12"; };
```

Comment:
```
unpacking source archive /build/v2.0.7.tar.gz
hash mismatch in fixed-output derivation '/nix/store/0s3ncb296vpdg8gni1c3wm3rff74f8bl-source':
  wanted: sha256:0zkfil7cxjip79wqvhljk1ifjq0cwxzx6wlxgp63b6wbagma0i12
  got:    sha256:1c01flf0xlm68dsv0a4n6bxglyj3x5mj8shmxglfpywbf74i3vnb
```

Reply:
same as above, should be fixed by [67d2f28](https://github.com/markuskowa/NixOS-QChem/pull/6/commits/67d2f28b597671907ff57e51cfb280605c2bef35)